src/backend/common/jit/BufferNodeBase.hpp

```diff
@@ -28,7 +28,9 @@ class BufferNodeBase : public common::Node {
     bool m_linear_buffer;
 
    public:
-    BufferNodeBase(af::dtype type) : Node(type, 0, {}) {}
+    BufferNodeBase(af::dtype type) : Node(type, 0, {}) {
+        // This class is not movable because of std::once_flag
```

Comment:
Is this comment a placeholder for `static_asserts` we have in other places because `BufferNodeBase is not movable` ?

Reply:
Yes